File: AMS.Web/src/app/models/batch-allocation/batch-matching-assessor-response.ts
import { OperationStatus } from "../shared/operation-status";
import { Assessor } from "../assessor-demographic/assessor";

export class BatchMatchingAssessorResponse
{
    //#region Constructor

    public BatchAllocationResponse()
    {
        this.Assessors = new Array<Assessor>();
        this.OperationStatus = new OperationStatus();
    }

    //#endregion Constructor

    //#region Class Properties

    Assessors:Array<Assessor>;

    OperationStatus:OperationStatus;

    //#endregion
}

File: AMS.API/DTO/BatchAllocation/BatchMatchingAssessorResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using ASSESSOR = AMS.API.DTO.Assessor;

namespace AMS.API.DTO.BatchAllocation
{
    public class BatchMatchingAssessorResponse
    {
        #region Constructor

        public BatchMatchingAssessorResponse()
        {
            Assessors = new List<ASSESSOR.Assessor>();
            OperationStatus = new OperationStatus();
        }
        #endregion Constructor

        #region Class Properties

        public List<ASSESSOR.Assessor> Assessors { get; set; }

        public OperationStatus OperationStatus { get; set; }

        #endregion
    }
}